modules/op-release-notes-1-6.adoc

```diff
@@ -274,4 +273,30 @@ Error from server (InternalError): Internal error occurred: failed calling webho
 // https://issues.redhat.com/browse/SRVKP-1783
 
 * The pod serving the `tkn` CLI will now be scheduled on nodes, based on the node selector and toleration limits specified in the `TektonConfig` custom resource.
-// https://issues.redhat.com/browse/SRVKP-1804
\ No newline at end of file
+// https://issues.redhat.com/browse/SRVKP-1804
+
+
+[id="release-notes-1-6-2_{context}"]
+== Release notes for {pipelines-title} General Availability 1.6.2
+
+[id="fixed-issues-1-6-2_{context}"]
+=== Fixed issues
+
+* Before this update, multiple instances of Tekton installer sets were created for a pipeline, after upgrading to Red Hat OpenShift Pipelines 1.6.1 from an older version. With this update, the Operator ensures that only one instance of each type of `TektonInstallerSet` exists after an upgrade.
+// https://issues.redhat.com/browse/SRVKP-1926
+
+* Before this update, all the reconcilers in the Operator used the component version to decide resource recreation, during an upgrade to Red Hat OpenShift Pipelines 1.6.1 from an older version. As a result, those resources were not recreated whose component versions did not change in the upgrade. With this update, the Operator uses the Operator version instead of the component version to decide resource recreation during an upgrade.
+// https://issues.redhat.com/browse/SRVKP-1928
+
+* Before this update, due to an update deadlock on the config maps, the Pipelines Webhook service was missing in the cluster, after an upgrade. With this update, a mechanism is added to disable webhook validation if the config maps are absent in the cluster. As a result, the Pipelines Webhook service persists in the cluster, after an upgrade.
+// https://issues.redhat.com/browse/SRVKP-1939
+
+* Before this update, cron jobs for auto-pruning got recreated, after any configuration change to the namespace. With this update, cron jobs for auto-pruning get recreated only if there is a relevant annotation change in the namespace.
+// https://issues.redhat.com/browse/SRVKP-1826
+
+* THe upstream version of Tekton Pipelines has been revised to `v0.28.3`, which has the following fixes:
+** Fix PipelineRun/TaskRun objects to label/annotation propagation.
```

Comment:
```suggestion
** Fix PipelineRun or TaskRun objects to label or annotation propagation.
```
Question: Is the use of "/" intentional here? Can this be replaced with "or"?

Reply:
Yes, / can be substituted by "or".